designs/administrator/4.6/column-management/index.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Column Management
+
+## Behavior
+Column management will live in the toolbar as a column icon on the Pods page on desktop. It will be hidden on mobile. Clicking on the icon will open a modal that lists all column names with checkboxes.
+![column-management-toolbar](img/column-management-toolbar.png)
+If the user is looking at “all projects”, then the checkbox next to “name” as well as “namespace”  in the modal will be shown as checked and disabled. Hovering over it will bring up a tooltip that explains that the columns may not be edited.
+![column-management-disabled-check](img/column-management-disabled-check.png)
```

Comment:
It sort of feel to me like the message should either include the count of total selected columns ("A total of 9 columns can be selected") or the word 'additional' or something should be in there ("You can select up to 7 additional columns"). Just a thought.

Reply:
Also maybe this tooltip could say why it can't be edited? "This column is required."?

Reply:
@itsptk The problem is that we are hiding `namespace` column when a project is selected and so we need to keep the alert consistent with both tables (since the user technically will only be seeing 8 columns at once when a specific project is selected). If we said `a total of 9 columns can be selected` then when you switch from `all projects` to `project` you'd get a random column that falls off due to having both `name` and `namespace` disabled when in `all projects`.

Tooltip could include information of why it can't be edited! That's a good idea.

Reply:
Gotcha, perhaps including "additional" in the message could still make it clear the the one/two required columns are not included in this count?